plaid/client.py

```diff
@@ -61,7 +61,7 @@ def store_access_token(func):
     def inner_func(self, *args, **kwargs):
         response = func(self, *args, **kwargs)
         if response.ok:
-            json_data = json.loads(response.content)
+            json_data = json.loads(response.content.decode())
```

Comment:
Is there any downside to using response.text instead of response .content?  

response.text returns the response decoded as unicode, so we don't have to manually decode.

The API is responding with JSON, so I would imagine this would be fine.